x-pack/legacy/plugins/reporting/server/lib/index.ts

```diff
@@ -4,11 +4,16 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-// @ts-ignore untyped module
-export { createTaggedLogger } from './create_tagged_logger';
 // @ts-ignore untyped module
```

Comment:
I think we should start looking at more of these untyped modules. I'll bring it up in our reporting sync.

Reply:
Making some type definitions might be easier first step than actually converting the modules to TS...

Reply:
++ to adding type definitions.